anta/catalog.py

```diff
@@ -39,8 +39,12 @@
 class AntaTestDefinition(BaseModel):
     """Define a test with its associated inputs.
 
-    test: An AntaTest concrete subclass
-    inputs: The associated AntaTest.Input subclass instance
+    Attributes
+    ----------
+    test
+        An AntaTest concrete subclass
+    inputs
+        The associated AntaTest.Input subclass instance
```

Comment:
Do we have a way to enforce this? I don't see any modifications of the Ruff config in pyproject or pre-commit so the current structure would still be allowed no?

Reply:
nope..